src/BlockNumber.js

```diff
@@ -36,20 +36,16 @@ export default function BlockNumber(props) {
   }, []);
 
   return (
-    <>
-      <Grid.Column>
-        <Card>
-          <Card.Content textAlign="center">
-            <Statistic
-              label={(finalized ? "Finalized" : "Current") + " Block Number"}
-              value={blockNumber}
-            />
-          </Card.Content>
-          <Card.Content extra>
-            <Icon name="time" /> {blockNumberTimer}
-          </Card.Content>
-        </Card>
-      </Grid.Column>
-    </>
+    <Grid.Column><Card>
+      <Card.Content textAlign="center">
+        <Statistic
+          label={(finalized ? "Finalized" : "Current") + " Block Number"}
+          value={blockNumber}
+        />
+      </Card.Content>
+      <Card.Content extra>
+        <Icon name="time" /> {blockNumberTimer}
+      </Card.Content>
+    </Card></Grid.Column>
   );
```

Comment:
This is even a better example of a stylistic change not related to the main point of the PR. If there were another PR open that actually changed logic in this section, they would needlessly conflict.

Reply:
I'm not saying this is something jimmy should change. Just pointing out a need for a style guide. In [the past](https://github.com/rchain-community/rchain-api/blob/master/package.json#L61) I've used airbnb's eslint rules effectively. I don't care which rules we use, but it would be helpful to standardize. Looks like polkadot-js team has done some related work https://github.com/polkadot-js/dev/blob/master/packages/dev/config/eslint.js . I'm not sure whether it addresses stuff like indentation though.

Reply:
You should drive this. Make a proposal for how we can all keep with the same JS formatting and we will do it. Right now I just auto format however visual studio code is set up.